modules/iyzicocheckoutform/views/templates/hook/infos.tpl

```diff
@@ -7,5 +7,17 @@
 <a href='https://www.iyzico.com' target='_blank'><strong>{l s="iyzico checkout form web sitesini" mod='iyzicocheckoutform'}</strong></a> 
 {l s="ziyaret edebilirsiniz." mod='iyzicocheckoutform'}</p>
 <br/>
-<p><strong>{l s="Not: https://merchant.iyzipay.com/settings adresinden API KEY ve SECRET KEY bilgilerinizi alabilirsiniz." mod='iyzicocheckoutform'}</strong></p>
+
+<p><strong>{l s="Note: You can get your API ID and Secret Key values from https://merchant.iyzipay.com/settings." mod='iyzicocheckoutform'}</strong></p>
 </div>
+{if $version.version_status == '1'}
+{if $version.version_status != ''}
```

Comment:
burayı anlamadım, zaten üstte bir if var ve birebir aynı değerin "1"e eşitliğini kontrol ediyor. 1'e eşitse boş'a eşit olamayacağına göre ikinci if'e gerek var mı?